ui/src/__mocks__/api.ts

```diff
@@ -1,14 +1,15 @@
 import { Org } from "../org";
 import { User } from "../user";
+import { Project } from "../project";
 
-type MockedResponse = Org | User | null;
+type MockedResponse = Org | Project | User | null;
 
 // just to give an idea of how we'd stub the api with other endpoints
 const userMock: User = {
   handle: "rafalca",
 };
 
-const radicleMock: Org = {
+export const radicleMock: Org = {
```

Comment:
Would be nice to find a more descriptive name for this mock as everything we produce is part of the radicle domain. Better to indicate that it is a mock project.